### ISSUE TITLE
blog example would be good

### ISSUE THREAD

**Issue by tunnckoCore** [yoshuawuyts/choo#177]
Maybe just as a reminder for me and you, because wanna try that choo :) Looks great, btw!

**Comment by yoshuawuyts** [comment]
There's the [choo-handbook](https://github.com/yoshuawuyts/choo-handbook) with several examples to help people get started. Is that what you mean?

**Comment by tunnckoCore** [comment]
I seen that, but no. I mean some sort of SPA blog. Found that today http://sennajs.com/examples/blog/

**Comment by yoshuawuyts** [comment]
woah, that blog page eats through CPU like it's nothing 🙁 

To be fair I think we've got a good number of examples in the repo already; I'm with you that perhaps something with markdown would be neat to have - but feel we should probably revisit all examples at some point in the near future (before a possible `v4`) to streamline the examples we got. But there's a whole bunch of work to do before this point too.

Anyway, heaps thanks for the suggestion! - We'll keep this in mind for when the examples get a revamp :grin: - closing for now as I'm on a bit of a mission to trim back the open issues. Thanks again!
